src/libs/migrations/KeyReportActionsByReportActionID.js

```diff
@@ -0,0 +1,52 @@
+import _ from 'underscore';
+import Onyx from 'react-native-onyx';
+import Log from '../Log';
+import ONYXKEYS from '../../ONYXKEYS';
+
+/**
+ * This migration updates reportActions data to be keyed by reportActionID rather than by sequenceNumber.
+ *
+ * @returns {Promise}
+ */
+export default function () {
+    return new Promise((resolve) => {
+        let shouldSkipMigration = false;
+        const connectionID = Onyx.connect({
+            key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
+            waitForCollectionCallbacks: true,
+            callback: (allReportActions) => {
```

Comment:
This does not seem to be working correctly. It's not all the reportActions ☹️

Reply:
Fixed. I had `waitForCollectionCallbacks` but it needed to be the singular `waitForCollectionCallback`